packages/shared/lib/core/profile/actions/active-profile/logout.ts

```diff
@@ -15,18 +15,19 @@ import {
     isLedgerProfile,
 } from '@core/profile'
 import { resetSelectedAccount } from '@core/account'
+import { isOnboardingLedgerProfile } from '@contexts/onboarding'
 
 /**
  * Logout from active profile
  */
-export function logout(clearActiveProfile: boolean = false, _lockStronghold: boolean = true): Promise<void> {
+export function logout(clearActiveProfile: boolean = true, _lockStronghold: boolean = true): Promise<void> {
     const { lastActiveAt, loggedIn, hasLoadedAccounts, type } = get(activeProfile)
 
     // (TODO): Figure out why we are using a promise here?
     return new Promise((resolve) => {
         if (get(isSoftwareProfile)) {
             _lockStronghold && lockStronghold()
-        } else if (isLedgerProfile(type)) {
+        } else if (isLedgerProfile(type) || get(isOnboardingLedgerProfile)) {
```

Comment:
This function is to logout of the active profile and is in the profile/actions/active-profile context. So if we need to clear down anything for onboarding then this should be outside of the logout function.